src/audio_core/sink/sdl2_sink.cpp

```diff
@@ -230,7 +230,10 @@ std::vector<std::string> ListSDLSinkDevices(bool capture) {
 
     const int device_count = SDL_GetNumAudioDevices(capture);
     for (int i = 0; i < device_count; ++i) {
-        device_list.emplace_back(SDL_GetAudioDeviceName(i, 0));
+        const char* name = SDL_GetAudioDeviceName(i, capture);
+        if (name != nullptr) {
```

Comment:
this can be collapsed inline, like:

```suggestion
        if (const char* name = SDL_GetAudioDeviceName(i, capture)) {
```